gammapy/detect/test_statistics.py

```diff
@@ -36,6 +35,31 @@
 CONTAINMENT = 0.8
 
 
+def _extract_array(array, shape, position):
+    """Helper function to extract parts of a larger array.
+
+    Simple implementation of an array extract function , because
+    `~astropy.ndata.utils.extract_array` introduces to much overhead.`
```

Comment:
typo: "to" -> "too"
There's a spurious backtick at the end of the line that should be removed.